pkgs/development/python-modules/aws-lambda-builders/default.nix

```diff
@@ -10,16 +10,16 @@
 
 buildPythonPackage rec {
   pname = "aws-lambda-builders";
-  version = "1.14.0";
+  version = "1.15.0";
   format = "setuptools";
 
   disabled = pythonOlder "3.7";
 
   src = fetchFromGitHub {
     owner = "awslabs";
     repo = "aws-lambda-builders";
-    rev = "v${version}";
-    sha256 = "sha256-ypzo0cYvP8LV74cQMzHIFDk3LH0bbEB4UxPxRuqe2fc=";
+    rev = "refs/tags/v${version}";
```

Comment:
out of curiosity, why this update is needed?

Reply:
It comes from here: https://github.com/NixOS/nixpkgs/commit/7f5915b04801f383a3504a011601e2fb09dfedf8